### ISSUE TITLE
groovy script

### ISSUE THREAD

**Issue by liufang732** [cobertura/cobertura#322]
can cobertura support code review for groovy script？ thanks

**Comment by jmrozanec** [comment]
@liufang732 yes, we are planning that for a future release. We may need some major refactorings but will enable us to support any JVM language. A proposal was [started in this fork](https://github.com/jmrozanec/cobertura-chocolate) You are welcome to share ideas and help us implement this.
